packages/apex-node/src/reporters/coverageReporter.ts

```diff
@@ -40,7 +43,10 @@ export const DefaultWatermarks: libReport.Watermarks = {
   lines: [50, 75]
 };
 
-export const DefaultReportOptions: Partial<reports.ReportOptions> = {
+export const DefaultReportOptions: Omit<
```

Comment:
dumb TS q: how'd you decide to use Omit vs Exclude?

Reply:
Omit: strip properties off of some type (think of an object)
Exclude: the type is a union type ex `x | y | z` and you want to disallow one of those types from the union.

Reply:
Report options is an object-looking interface and there are 2 props we don't support.

Reply:
CoverageReportFormats is a bunch of strings in a union